operator/e2e/e2e_test.go

```diff
@@ -91,9 +91,6 @@ func TestE2E(t *testing.T) {
 
 				cmd := exec.Command("bash", "-c", "envsubst < e2e/testdata/git-auth-nonroot/* | kubectl apply -f -")
 				require.NoError(t, run(cmd))
-				t.Cleanup(func() {
-					_ = run(exec.Command("kubectl", "delete", "-f", "e2e/testdata/git-auth-nonroot"))
-				})
```

Comment:
This was deleting the secret needed by the operator in order to poll the repo, preventing the stack from reconciling / getting deleted.